reference/6/Microsoft.PowerShell.Core/Remove-Module.md

```diff
@@ -205,6 +205,20 @@ You can pipe module names and module objects to **Remove-Module**.
 This cmdlet does not generate any output.
 
 ## NOTES
+When removing a module, there is an event on the module that will execute. This allows a module to react to being removed and maybe free up resources. Example:
```

Comment:
Please use [semantic line breaks](http://sembr.org/) which basically means putting each sentence on a separate line.  Rendering will put it back together in paragraphs.